tools/src/scie_pants/configure_pants.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2022 Pants project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+import os
+import shutil
+import subprocess
+import sys
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import NoReturn
+
+from packaging.version import Version
+
+from scie_pants.log import fatal, info, init_logging, warn
+from scie_pants.pants_version import (
+    determine_latest_stable_version,
+    determine_sha_version,
+    determine_tag_version,
+)
+from scie_pants.ptex import Ptex
+
+
+def prompt(message: str, default: bool) -> bool:
+    raw_answer = input(f"{message} ({'Y/n' if default else 'N/y'}): ")
+    answer = raw_answer.strip().lower()
+    if not answer:
+        return default
+    return answer in ("y", "yes")
+
+
+def prompt_for_pants_version(pants_config: Path) -> bool:
+    warn(
+        f"The `pants.toml` at {pants_config} has no `pants_version` configured in the `GLOBAL` "
+        f"section."
+    )
+    return prompt(f"Would you like set `pants_version` to the latest stable release?", default=True)
+
+
+def prompt_for_pants_config() -> Path | None:
+    cwd = os.getcwd()
+    buildroot = Path(cwd)
+    if shutil.which("git"):
+        result = subprocess.run(
+            args=["git", "rev-parse", "--show-toplevel"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.DEVNULL,
+        )
+        if result.returncode == 0:
+            buildroot = Path(os.fsdecode(result.stdout.strip()))
```

Comment:
This could conceivably be annoying for some strange case where you want pants.toml, for some reason, set up under the ambient `.git`. It seems to me that situation is almost by definition involving a user who doesn't need one step setup! That said, it may be good to follow up here and provide an extra level of prompting that allows build root selection override. I was hoping to keep this simple for initial release / announce tomorrow though and hopefully let others with opinions about UX chime in by doing - this stuff needs more contributors.

Reply:
The UX for all this can be read from the new IT logs:
https://github.com/pantsbuild/scie-pants/actions/runs/3728396083/jobs/6323372959#step:6:184

Probably easier to grok just trying for yourself. I just copy `dist/scie-pants-XYZ` to `~/bin/pants` and go play around executing `pants`.

Reply:
That `pants.toml` under `.git` case sounds atrocious, and I am fine making it hard to do for now.